sky/clouds/service_catalog/gcp_catalog.py

```diff
@@ -292,7 +292,9 @@ def get_instance_type_for_accelerator(
 
     if acc_name in _ACC_INSTANCE_TYPE_DICTS:
         df = _df[_df['InstanceType'].notna()]
-        instance_types = _ACC_INSTANCE_TYPE_DICTS[acc_name][acc_count]
+        instance_types = _ACC_INSTANCE_TYPE_DICTS[acc_name].get(acc_count, None)
+        if instance_types is None:
+            return None, []
```

Comment:
Do we still need catalog schema bump if we use `.get(acc_count, None)`? won't it work with v5?

Reply:
No, since this changes will not affect the old SkyPilot version, we have to bump the catalog version to ensure old skypilot release can work. We add this `get` for future proof. : )

Reply:
Oh nvm, I think I understand - we still need the catalog version bump for back compat.